Source/Libraries/FaultData/DataReaders/EMAXReader.cs

```diff
@@ -114,8 +115,19 @@ public void ExportToCOMTRADE()
 
                     comtradeAnalog.Multiplier = (max - min) / (2 * short.MaxValue);
                     comtradeAnalog.Adder = (max + min) / 2.0D;
```

Comment:
Since we adjusted the ratios, will we need to adjust the adder/multiplier? Sorry, I don't fully understand how these fields interact.

Reply:
The adder/multiplier are only used during encoding/decoding. They have no direct relationship to the primary/secondary fields. Furthermore, this PR doesn't actually change the PT/CT ratio, but just reduces the fraction.

Reply:
Okay, thank you. I just wanted to be sure since I wasn't 100% certain.